open-company/open-company-web
# Comment action buttons and NEW label

Cards:
- https://trello.com/c/Omnv5vlV
- https://trello.com/c/ztlYDj9c

To test NEW label:
- user A go to AP
- user B add a new comment to a post user A sees in AP
- user A sees NEW label appear
- user B adds a new comment
- user A still sees NEW label
- user A expand post
- [x] only new comments have the NEW badge besides them
- user A go back to AP and back in the post
- [x] all comments have no NEW label
- user B adds a new comment
- [x] user A sees the new comment appear and it has NEW label besides it
- user A goes back to AP
- user B adds a comment to the post
- user A expand the post
- [x] the new comment has NEW label
- user B adds a new comment
- [x] the last and the but last comments have NEW label besides them

To test layout:
- [ ] make sure the comments layout properly as in design
- [x] make sure the add comment box is like in design (check with enabled/disabled comment button)
- [x] make sure reply comment field is correct
- [x] make sure edit comment field is correct
- [x] check the rest of the design (refer to the trello card for more details on what changed)
